### PR TITLE
yabai: init at 2.1.2

### PR DESCRIPTION
**Pull request by jacereda**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/os-specific/darwin/yabai/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  installPhase = ''
+    install -d $out/bin
+    cp bin/yabai $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A tiling window manager for macOS based on binary space partitioning";
+    homepage = https://github.com/koekeishiya/yabai;
+    platforms = platforms.darwin;
```
Comment (Lassulus) [inline pkgs/os-specific/darwin/yabai/default.nix]: a maintainer would be cool

File: pkgs/os-specific/darwin/yabai/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+, IOKit
+, ScriptingBridge
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yabai";
+  version = "2.1.2";
+  src = fetchFromGitHub {
+    owner = "koekeishiya";
+    repo = "yabai";
```
Comment (ryantm) [inline pkgs/os-specific/darwin/yabai/default.nix]:
```suggestion
    repo = pname;
```

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by bkase** [comment]
@ryantm is this ready to go now?

**Comment by Lassulus** [comment]
Merging without a maintainer is not preferred. Since there is no one who will update it or can be pinged for reviews.

**Comment by jacereda** [comment]
I'm seriously considering switching to Linux in the not too distant future, so I'm not a good candidate for maintaining this.

**Comment by Lassulus** [comment]
@cmacrae sounds like a good idea :+1:

**Comment by cmacrae** [comment]
@ryantm @Lassulus I opened this yesterday ☝🏻😊

**Pull request closed without merging**